Removed 5 from HTML (#19745)
mdn/content: files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md

---
{{DefaultAPISidebar("Gamepad API")}}

> HTML provies the necessary components for rich, interactive game development. Technologies like `<canvas>`, WebGL, `<audio>`, and `<video>`, along with JavaScript implementations support tasks that provide similar, if not the same, features as native code. The Gamepad API is a way for developers and designers to access and use gamepads and other game controllers.

Reviewer: ```suggestion
HTML provides the necessary components for rich, interactive game development. Technologies like `<canvas>`, WebGL, `<audio>`, and `<video>`, along with JavaScript implementations, support tasks that provide similar, if not the same, features as native code. The Gamepad API allows developers and designers to access and use gamepads and other game controllers.
```
